moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -449,8 +460,13 @@ void planning_scene_monitor::CurrentStateMonitor::tfCallback()
         continue;
       }
 
+      // TODO(JafarAbdi): Replace with insert_or_assign once we have C++17 support
```

Comment:
Could you open an issue for C++17? ;) I'd be fully on board with this if it doesn't come with unwanted side effects.